src/utilities/activityContent.test.ts

```diff
@@ -49,12 +49,6 @@ describe("activityContent", () => {
         attachmentType: "video/webm",
         mediaType: "Video",
       },
-      {
```

Comment:
our activityContent.ts file does not support this type, so it was falling into the default type, "Images" and the test was failing.